src/test/java/jewellery/inventory/integration/OrganizationCrudIntegrationTest.java

```diff
@@ -0,0 +1,89 @@
+package jewellery.inventory.integration;
+
+import static jewellery.inventory.helper.OrganizationTestHelper.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import io.micrometer.common.lang.Nullable;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import jewellery.inventory.dto.request.OrganizationRequestDto;
+import jewellery.inventory.dto.response.OrganizationResponseDto;
+import jewellery.inventory.dto.response.ProductResponseDto;
+import jewellery.inventory.model.Organization;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+
+class OrganizationCrudIntegrationTest extends AuthenticatedIntegrationTestBase {
+  private String getBaseOrganizationsUrl() {
+    return "/organizations";
+  }
+
+  private String getOrganizationByIdUrl(UUID id) {
+    return "/organizations/" + id;
+  }
+
+  private Organization organization;
+  private OrganizationRequestDto organizationRequestDto;
+
+  @BeforeEach
+  void setUp() {
+    organization = getTestOrganization();
+    organizationRequestDto = getTestOrganizationRequest();
+  }
+
+  @Test
+  void getOrganizationsSuccessfully() {
+    ResponseEntity<List<ProductResponseDto>> response =
+        this.testRestTemplate.exchange(
+            getBaseOrganizationsUrl(), HttpMethod.GET, null, new ParameterizedTypeReference<>() {});
+
+    assertEquals(response.getStatusCode(), HttpStatusCode.valueOf(200));
+    assertNotNull(response.getBody());
```

Comment:
This is not a meaningful assertion. I know we are doing a lot of this kind of assertions in other tests but we should be more specific with our assertions. We should rework our tests setup. Ignore this for now. We will do it in a separate ticket